src/functionalTest/resources/features/F-204 - Get NoC Questions/common/204_Submit_ManageOrgs_Event.td.json

```diff
@@ -0,0 +1,24 @@
+{
+  "_guid_" : "204_update_org_policies",
+  "_extends_": "update_org_policies",
+
+  "specs" : [
+    "to update the applicant org policy"
+  ],
+
+  "user": {
+    "username": "befta.caseworker.caa@gmail.com",
+    "password": "[[$CCD_CASEWORKER_AUTOTEST_PASSWORD]]"
+  },
```

Comment:
As before: try: 
```
  "user": {
    "_extends_": "ACA_Users_Caseworker_CAA"
  },
```

Reply:
Switching to use Richard account as CAA shouldn't have access to these extra events. 

```
  "user": {
    "_extends_": "ACA_Users_Richard"
  },
```